apps/admin-server/package.json

```diff
@@ -15,6 +15,7 @@
     "@openstad-headless/raw-resource": "file:../../packages/raw-resource",
     "@openstad-headless/likes": "file:../../packages/likes",
     "@openstad-headless/ui": "file:../../packages/ui",
+    "@openstad-headless/form": "file:../../packages/form",
```

Comment:
Ik vraag me af of de /ui en form folder geinclude moet worden in dit project. Uiteindelijk tenzij de admin deze direct gebruikt zou dit alleen in de widget package.json. hoeven te zitten dacht ik. Is geen ramp, maar als het niet nodig is, dan kan dit er beter uit gelaten worden